app/src/behaviors/behavior_turbo_key.c

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#define DT_DRV_COMPAT zmk_behavior_turbo_key
+
+#include <zephyr/device.h>
+#include <drivers/behavior.h>
+#include <zephyr/logging/log.h>
+
+#include <zmk/hid.h>
+#include <zmk/event_manager.h>
+#include <zmk/events/keycode_state_changed.h>
+#include <zmk/behavior.h>
+#include <zmk/behavior_queue.h>
+
+LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
+
+struct behavior_turbo_config {
+    int tap_ms;
+    int wait_ms;
+    int toggle_term_ms;
+    const struct zmk_behavior_binding binding;
+};
+
+struct behavior_turbo_data {
+    uint32_t position;
+    bool is_active;
+    bool is_pressed;
+
+    int32_t press_time;
+
+    int tap_ms;
+    int wait_ms;
+    struct zmk_behavior_binding binding;
+
```

Comment:
I don't see any reason this is duplicated in the data, and in the config. Should just be in the config?